Extensions/PlatformBehavior/platformerobjectruntimebehavior.ts

```diff
@@ -435,16 +445,15 @@ namespace gdjs {
           runtimeScene.getGame().getInputManager().isKeyPressed(RSHIFTKEY) ||
           runtimeScene.getGame().getInputManager().isKeyPressed(SPACEKEY));
       if (this._canJump && this._jumpKey) {
-        this._jumping = true;
+        this._state = PlatformerObjectRuntimeBehavior.State.Jumping;
         this._canJump = false;
         this._timeSinceCurrentJumpStart = 0;
         this._jumpKeyHeldSinceJumpStart = true;
 
+        //FIXME can't be Jumping and OnFloor at the same time, need a fix and a test
         //this._isOnFloor = false; If floor is a very steep slope, the object could go into it.
-        this._isOnLadder = false;
         this._currentJumpSpeed = this._jumpSpeed;
         this._currentFallSpeed = 0;
-        this._isGrabbingPlatform = false;
       }
```

Comment:
It relied on the superposition of OnFloor and Jumping to fix an issue with steep slopes. So, there certainly is a regression. I need check this, write a test and find another way to fix it.

Reply:
Try to make a floor that is rotated so that it's very steep. I think the issue that then when you would jump from it, you would get stuck into the floor, if you press the key to move toward the floor, or you would stop the jump almost immediately.

Reply:
I could reproduce on this PR, but it's fixed by the other PR:
https://github.com/4ian/GDevelop/pull/2396
So, I think this PR can be forgotten.